src/components/pages/Deals.js

```diff
@@ -1,29 +1,83 @@
 import React, { useState, useEffect } from 'react';
-import PropTypes from 'prop-types';
+import axios from 'axios';
 import ItemList from '../items/ItemList';
+import Search from '../layout/Search';
+import PageBtns from '../layout/PageBtns';
 
-const Deals = (props) => {
-  const { fetchData } = props;
+const Deals = () => {
   const [deals, setDeals] = useState(null);
+  const [searchFail, setSearchFail] = useState(false);
+  const [pageSize] = useState(6);
```

Comment:
Since you don't ever update the page size, you can set this as a constant. And since you are also using this value in **Home.js**, you can export the value from a `/src/utils/constants.js` (or similar) file

Reply:
fixed in e968a49